cordova-plugin-one/README.md

```diff
@@ -10,9 +10,12 @@ $ cordova plugin add cordova-plugin-one
 ```
 ## Usage
 ### Initialization
-To initialize the ONE Cordova Plugin, call the following method:
+To initialize the ONE Cordova Plugin, fetch the `One` instance from the window and call the following method:
```

Comment:
What's the reason for the change? Is calling from the window required?

Reply:
By default, yes.  We will have to create an Ionic Native Wrapper in order to fetch ONE natively.

Reply:
Created MOB-1301 to track